.travis.yml

```diff
@@ -9,11 +9,17 @@ env:
 services:
   - postgresql
   - redis-server
+  - docker
 install:
   - pip install -r requirements.txt
 script:
   - flake8
   - python manage.py test
+  - docker build -t pyslackers/website:$TRAVIS_BRANCH .
 matrix:
   allow_failures:
-    - "3.7-dev"
+    python: "3.7-dev"
```

Comment:
If it runs (as it did) and their docs show it - I am not sure we should care about their linter since it seems out of date. I prefer it as it's more flexible (more than 1 py version)